dali/operators/image/crop/crop_mirror_normalize.cc

```diff
@@ -22,14 +22,15 @@
 namespace dali {
 
 DALI_SCHEMA(CropMirrorNormalize)
-  .DocStr(R"code(Perform fused cropping, normalization, format conversion
+  .DocStr(R"code(Performs fused cropping, normalization, format conversion
 (NHWC to NCHW) if desired, and type casting.
-Normalization takes input image and produces output using formula::
+
+Normalization takes the input image and produces the output by using the following formula::
 
   output = (input - mean) / std
 
-Note that not providing any crop argument will result into mirroring and
-normalization only.
+.. note::
+    If you do not provide a crop argument, only  mirroring and normalization will occur.
```

Comment:
Again, we've decided not to address the reader directly.
```suggestion
    If no cropping method is specified, only mirroring and normalization will occur.
```

Reply:
Done